.travis.yml

```diff
@@ -22,6 +22,8 @@ matrix:
       env: WP_VERSION=3.7.11
     - php: 5.3
       env: WP_VERSION=3.7.11 DEPLOY_BRANCH=master
+    - php: 7.1
+      env: WP_VERSION=latest SAVE_COMMENTS_DISABLED=1
```

Comment:
I left the tag in the "bootstrap.feature" test as a sort of reminder/comment. That and the passing of `WP_CLI_PHP_ARGS` in `FeatureContext::get_process_env_variables()` are unnecessary now - should I remove them?